rohit/widgets/src/App.js

```diff
@@ -34,22 +35,9 @@ const options = [
 ];
 
 const App = () => {
-
-  const [selected, setSelected] = useState(options[0]);
-  const [showDropdown, setShowDropdown] = useState(true);
-  
   return (
     <div>
-      <button className='ui red button' onClick={() => setShowDropdown(!showDropdown)}>
-        Toggle Dropdown
-      </button>
-      {showDropdown ?
-        <Dropdown
-          selected={selected}
-          onSelectedChange={setSelected}
-          options={options}
-        /> : null
-      }
+      <Translate options={options}  />
```

Comment:
The `Translate` component does not expect any prop. 
```suggestion
      <Translate />
```